nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/components/explorer/grid-list/registry/nf-registry-grid-list-viewer.html

```diff
@@ -121,6 +121,38 @@
                                     </td-step>
                                 </td-steps>
                             </div>
+                            <div *ngIf="droplet.bundleType" id="nifi-registry-explorer-grid-list-viewer-droplet-container-details-change-log">
+                                <td-steps mode="vertical">
+                                    <td-step label="Version {{snapshotMeta.version}} ({{nfRegistryService.getHumanReadableSize(snapshotMeta.contentSize)}}) - {{snapshotMeta.timestamp | amTimeAgo}}" sublabel="by {{snapshotMeta.author}}"
+                                             *ngFor="let snapshotMeta of droplet.snapshotMetadata; let i = index"
+                                             [active]="i === 0 ? true : false">
+                                        <div fxLayout="column" fxLayoutAlign="space-between stretch">
+                                            <div fxLayout="row" class="mat-body-3">
+                                                <p>
+                                                Build Information<br />
+                                                &nbsp;&nbsp;&nbsp;&nbsp;<b>NiFi API version</b>: {{snapshotMeta.systemApiVersion}}<br />
+                                                &nbsp;&nbsp;&nbsp;&nbsp;<b>Build branch</b>: {{snapshotMeta.buildInfo.buildBranch}}<br />
+                                                &nbsp;&nbsp;&nbsp;&nbsp;<b>Build flags</b>: {{snapshotMeta.buildInfo.buildFlags}}<br />
+                                                &nbsp;&nbsp;&nbsp;&nbsp;<b>Build revision</b>: {{snapshotMeta.buildInfo.buildRevision}}<br />
+                                                &nbsp;&nbsp;&nbsp;&nbsp;<b>Build tag</b>: {{snapshotMeta.buildInfo.buildTag}}<br />
+                                                &nbsp;&nbsp;&nbsp;&nbsp;<b>Build tool</b>: {{snapshotMeta.buildInfo.buildTool}}<br />
+                                                &nbsp;&nbsp;&nbsp;&nbsp;<b>Built by</b>: {{snapshotMeta.buildInfo.builtBy}}<br />
+                                                &nbsp;&nbsp;&nbsp;&nbsp;<b>Built on</b>: {{(snapshotMeta.buildInfo.built/1000 | amFromUnix) | amDateFormat:'MMM-DD-YYYY'}} at {{(snapshotMeta.buildInfo.built/1000 | amFromUnix) | amDateFormat:'h:mm A'}}
```

Comment:
Is the `MMM-DD-YYYY` format common elsewhere in the application? I recommend using the ISO 8601 format of `YYYY-MM-DD`.

Reply:
It's also used for the date at which the version has been added in the NiFi Registry, but agree that using the other format is better